src/shared/utilities/download.ts

```diff
@@ -36,9 +42,18 @@ export async function downloadExportResultArchive(
             const chunkData = chunk.binaryPayloadEvent
             if (chunkData.bytes) {
                 buffer.push(chunkData.bytes)
+                totalDownloadBytes += chunkData.bytes?.length
             }
         }
     }
 
     fs.outputFileSync(toPath, Buffer.concat(buffer))
+    telemetry.codeTransform_logApiLatency.emit({
+        codeTransformApiNames: 'ExportResultArchive',
```

Comment:
when does `codeTransform_logApiLatency` emit  while jobs are running?

Reply:
This metric is meant to emit right after we make a call to the backend service, to track the latency felt by the user. This is a special case because the backend returns a stream, which we read to the end (as part of downloading the zip) before we log the latency